test/test_cutout_termination.smk

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
```

Comment:
Many thanks for the test example! :)
Do you think it shall be included or can we remove it? [less material to maintain]